manifests/02b_argocd-secret.yaml

```diff
@@ -7,20 +7,20 @@ metadata:
 type: Opaque
 stringData:
   # bcrypt hash of the string "password"
-  admin.password: $2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W
+  admin.password: ""
```

Comment:
Ever key in this file can be commented out. Basically, we should start out with an empty secret (vs. a secret with empty values).

During API server startup, after retrieving the `argocd-secret`, if the keys `admin.password` and `server.secretkey ` are *missing*, the API server should populate it. This is different than checking that it is an empty string.

The reason we want it to be missing vs empty string is because a subsequent `kubectl apply` of the yaml will clobber the values that the API server generated.